src/modules/shop/domain/Products.test.ts

```diff
@@ -0,0 +1,15 @@
+import {Products} from './Products';
+
+describe('Testing Products', () => {
+  it('Get Products List', async () => {
+    const products = new Products();
+
+    const productsList = await products.get();
+
+    expect(productsList.success).toBe(true);
+    expect(!!productsList.statusCode && productsList.statusCode === 200).toBe(
```

Comment:
Não é uma boa usar magic numbers, cria uma variável para status code 200, e usa aqui.
```suggestion
    expect(!!productsList.statusCode && productsList.statusCode === SUCESS_CODE).toBe(
```